### DISCUSSION TITLE
Preview and download control？

### DISCUSSION THREAD

**Discussion by shenzgang** [minio/minio#13761] [Q&A]
I created two buckets, each with the permission of public,Read and Write, to access the image directly, one is to download, the other is to display in the browser.  How do I control downloads and previews?

**Comment by harshavardhana** [comment]
Not sure what this question means clearly, can you elaborate more?

**Reply by shenzgang** [reply]
hello, I have enabled permanent access for the specified bucket. I want to access the image directly in the browser through the connection, but some buckets download the image directly. I just want to open it in the browser, not download it.  How do I control whether I download it or open it directly in the browser?

**Reply by harshavardhana** [reply]
> hello, I have enabled permanent access for the specified bucket. I want to access the image directly in the browser through the connection, but some buckets download the image directly. I just want to open it in the browser, not download it. How do I control whether I download it or open it directly in the browser?

You need to set the correct content-type

**Reply by shenzgang** [reply]
thanks, i get it!
